web/components/feed/feed-comments.tsx

```diff
@@ -465,6 +484,18 @@ export function CommentInput(props: {
                 )}
               </Col>
             </Row>
+            <Row>
+              {!user && (
+                <button
+                  className={
+                    'btn btn-outline btn-sm text-transform:capitalize mt-2'
```

Comment:
We should discuss this, but I believe it's common practice to not capitalize any words other than the first in UI. I think we should go with this style.

E.g. Sign in to comment

Reply:
Sounds good to me